podman/api_connection.py

```diff
@@ -87,22 +94,16 @@ def post(self, path, params=None, headers=None):
                             body=data,
                             headers=headers)
 
-    def request(self,
-                method,
-                url,
-                body=None,
-                headers=None,
-                *,
-                encode_chunked=False):
+    def request(
+        self, method, url, body=None, headers=None, *, encode_chunked=False
```

Comment:
The previous way was awful, but I don't think we should stick to aligning with the opening delimiter rather than doing it this way.

If we must split because line length (though we're using 100 instead of <80)


```
def request(self, method, url, body=None, headers=None, *, 
            encode_chunked=False):
```

Reply:
Thanks for the review @mwhahaha. Black formatted the code this way. I thinks is more readable, tho.